.gitmodules

```diff
@@ -0,0 +1,9 @@
+[submodule "documentation/src/docs/pravega-operator"]
+       path = documentation/src/docs/pravega-operator
+       url = https://github.com/pravega/pravega-operator.git
+[submodule "documentation/src/docs/flink-connectors"]
```

Comment:
Interestingly, Flink connectors repo has another reference to Pravega, so we are creating a kind of circular reference here.

Reply:
@crazyzhou thinks he might be able to remove his pravega submodule from the flink-connectors repo. In the meantime, it shouldn't be a problem unless someone does a recursive submodule init, then I might expect an infinite loop of checkouts :)